src/LPUtf8StrMarshaler.cs

```diff
@@ -30,7 +30,7 @@
 
 namespace SDL2
 {
-	internal unsafe class LPUtf8StrMarshaler : ICustomMarshaler
+	public unsafe class LPUtf8StrMarshaler : ICustomMarshaler
```

Comment:
Because this is a generally useful class; there's nothing SDL-specific about it.